srcpkgs/PackageKit-qt/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'PackageKit-qt'
+pkgname=PackageKit-qt
+version=1.1.0
+revision=1
+archs="x86_64"
```

Comment:
why only x86_64 (for both)?

Reply:
I guess I'm just not sure if it will compile for other architectures, and I don't know how to figure out which ones are friendly and which ones aren't.

Reply:
try removing the line, it will probably work on all supported archs